### PR TITLE
Maven setup

### PR DESCRIPTION
**Pull request by EliasJorgensen**
The following has been done:

- Maven setup with build config and dependencies
- Liquibase setup, configured to run migrations on build
- H2 setup, see DataSource URL in pom file
- JDBI test Dao, showing off how to use the JDBI Declarative API

### CODE REVIEW

File: src/test/java/com/noticemedan/cinema/mainTest.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package com.noticemedan.cinema;
+
+public class mainTest {
```
Comment (EliasJorgensen) [inline src/test/java/com/noticemedan/cinema/mainTest.java:3]: BLOCKER Add some tests to this class. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2187)

File: src/main/java/com/noticemedan/cinema/Application.java [hunk 2]
```diff
@@ -1,8 +1,22 @@
...
 public class Application {
     public static void main(String[] args) {
 
+        Jdbi jdbi = Jdbi.create("jdbc:h2:~/cinemadb/cinema;AUTO_SERVER=true");
+        jdbi.installPlugin(new H2DatabasePlugin());
+        jdbi.installPlugin(new SqlObjectPlugin());
+
+        TestDao dao = jdbi.onDemand(TestDao.class);
+
         System.out.println("Hey");
```
Comment (EliasJorgensen) [inline src/main/java/com/noticemedan/cinema/Application.java:19]: MAJOR Replace this use of System.out or System.err by a logger. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)

File: src/main/java/com/noticemedan/cinema/Application.java [hunk 3]
```diff
@@ -1,8 +1,22 @@
...
+        System.out.println(Arrays.deepToString(dao.listTests().toArray()));
```
Comment (EliasJorgensen) [inline src/main/java/com/noticemedan/cinema/Application.java:20]: MAJOR Replace this use of System.out or System.err by a logger. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)

**Comment by EliasJorgensen** [comment]
SonarQube analysis reported 3 issues
* ![BLOCKER][BLOCKER] 1 blocker
* ![MAJOR][MAJOR] 2 major

Watch the comments in this conversation to review them.

[BLOCKER]: https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: pom.xml [hunk 4]
```diff
@@ -8,5 +8,84 @@
...
+            <artifactId>slf4j-log4j12</artifactId>
+            <version>1.7.21</version>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
 
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
```
Comment (MrMatten) [inline pom.xml:61]: Hvad gør den her?

**Review by MrMatten — APPROVED** [review]
Ser fint ud

File: pom.xml (hunk 4 above)
Comment (EliasJorgensen) [reply pom.xml:61]: Den compiler projektet. Den bruger JDK 1.5 som default, så jeg satte den til at bruge 1.9 i stedet.

**Comment by EliasJorgensen** [comment]
Merging

**Pull request merged**